### PR TITLE
♻️Maintenance: Mypy on servicelib

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

- relates to https://github.com/ITISFoundation/osparc-simcore/issues/6115

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: packages/service-library/src/servicelib/aiohttp/monitoring.py [hunk 1]
```diff
@@ -107,18 +107,18 @@
 # http_request_latency_seconds_created{app_name="simcore_service_webserver",endpoint="/metrics",method="GET"} 1.641806371709292e+09
 
 
-kREQUEST_COUNT = f"{__name__}.request_count"
-kINFLIGHTREQUESTS = f"{__name__}.in_flight_requests"
-kRESPONSELATENCY = f"{__name__}.in_response_latency"
+kREQUEST_COUNT = f"{__name__}.request_count"  # noqa: N816
```
Comment (GitHK) [inline packages/service-library/src/servicelib/aiohttp/monitoring.py:110]: maybe it requires a type to be `Final[str]` as well? I guess N816 is the lowercase "k"

File: packages/service-library/src/servicelib/fastapi/http_client_thin.py [hunk 2]
```diff
@@ -147,13 +147,16 @@ async def request_wrapper(zelf: "BaseThinClient", *args, **kwargs) -> Response:
...
                     with attempt:
-                        r: Response = await request_func(zelf, *args, **kwargs)
-                        return r
+                        return await request_func(zelf, *args, **kwargs)
             except HTTPError as e:
                 if isinstance(e, PoolTimeout):
                     _log_pool_status(zelf.client, "pool timeout")
                 raise ClientHttpError(error=e) from e
 
+            # NOTE: this satisfies mypy, both eexceptions from retrial are HTTPError
```
Comment (GitHK) [inline packages/service-library/src/servicelib/fastapi/http_client_thin.py:156]:
```suggestion
            # NOTE: this satisfies mypy, both exceptions from retrial are HTTPError
```

**Review by GitHK — APPROVED** [review]
👍

File: packages/service-library/src/servicelib/aiohttp/monitoring.py (hunk 1 above)
Comment (sanderegg) [reply packages/service-library/src/servicelib/aiohttp/monitoring.py:110]: yes that relates to mixing upper/lower case

**Pull request merged**